batavia/types/Bool.js

```diff
@@ -515,6 +515,12 @@ Bool.prototype.__lshift__ = function(other) {
             return new types.Int(0)
         }
     } else if (types.isinstance(other, types.Int)) {
+        if(other.valueOf() < 0) {
```

Comment:
The linter is *probably* going to yell at you about having spaces after `if` (here and below).